### DISCUSSION TITLE
Web-Stories feature

### DISCUSSION THREAD

**Discussion by Aditya-eddy** [keploy/keploy#1790] [Ideas]
Hey Mentors,
I have created webstories feature. Webstories will allow us to deliver the data more efficiently and in a more new interactive way.
Here I have created a component which will allow us to reuse the component for displaying webstories.
Please give me feedback on the webstories upon which i can work on improving it!!
cc- @nehagup , @Hermione2408 , @charankamarapu ,
Video:-
[Screencast from 08-04-24 11:23:37 AM IST.webm](https://github.com/keploy/keploy/assets/70089590/efb43b9e-3888-4d25-81ee-1e69bb0be965)

**Comment by nehagup** [comment]
This is good @Aditya-eddy ! design-wise looks good, it'll be good to add keploy navbar at the top and also theme colors in the progress bar and other elements. Can you please submit the prototype in a PR? 

P.S. You may have to plan the dark-mode early on.

**Comment by Aditya-eddy** [comment]
Hey @nehagup will be working on the navbar and other elements as you said. Will be submitting the pr for it soon!

**Comment by Aditya-eddy** [comment]
Hey mentors , please give me feedback me on this:
1. should we create a  nav bar in /webstories/[slug]. Personally i think it would look good without the Navbar. Also in the [ref](https://www.lambdatest.com/web-stories/exciting-features-of-selenium-4). It is without navbar. I will post screenshot with navbar and without  navbar:
![Screenshot from 2024-04-12 10-43-20](https://github.com/keploy/keploy/assets/70089590/b2923515-5fc5-43c6-89de-ce82e0c3e179)
![Screenshot from 2024-04-12 10-42-47](https://github.com/keploy/keploy/assets/70089590/ce6bbd3e-0464-4d22-b3ab-830b5002ec06)
cc- @nehagup , @Hermione2408
